src/value/owned.rs

```diff
@@ -68,6 +68,38 @@ impl OwnedSymbolToken {
             source,
         }
     }
+
+    /// Decorates the [`OwnedSymbolToken`] with text.
+    pub fn with_text<T: Into<Rc<str>>>(mut self, text: T) -> Self {
+        self.text = Some(text.into());
+        self
+    }
+
+    /// Decorates the [`OwnedSymbolToken`] with a local ID.
+    pub fn with_local_sid(mut self, local_sid: SymbolId) -> Self {
+        self.local_sid = Some(local_sid);
+        self
+    }
+
+    /// Decorates the [`OwnedSymbolToken`] with an [`OwnedImportSource`].
+    pub fn with_source<T: Into<Rc<str>>>(mut self, table: T, sid: SymbolId) -> Self {
+        self.source = Some(OwnedImportSource::new(table, sid));
+        self
+    }
+}
```

Comment:
Are these names idiomatic?  @zslayton might have an opinion.

Reply:
These names are idiomatic, but I'd really prefer to spell out the word `token` in `local_sid_tok` and `text_tok` below.